client/app/bundles/course/announcements/components/misc/AnnouncementsDisplay.tsx

```diff
@@ -56,21 +56,15 @@ const AnnouncementsDisplay: FC<Props> = (props) => {
     setShavedAnnouncements(announcements);
   }, [announcements]);
 
-  const handleSearchBarChange = (
-    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>,
-  ): void => {
-    if (event.target.value.trim() === '') {
+  const handleSearchBarChange = (keyword: string): void => {
```

Comment:
Do you think we need to trim the end of the keyword? I understand the `trims` in `TextField` only trims the start of the string

Reply:
Thanks for catching this. Yes, `trims` only trims the start of the string `onChange`. I will add the additional `.trim()`s.